test/alternator/test_manual_requests.py

```diff
@@ -100,8 +102,10 @@ def test_too_large_request(dynamodb, test_table):
 #    Content-Length header or chunked encoding (reproduces issue #8196).
 # 2. The client should be able to recognize this error as a 413 error, not
 #     some I/O error like broken pipe (reproduces issue #8195).
-@pytest.mark.xfail(reason="issue #8195, #8196")
+@pytest.mark.xfail(reason="issue #8196")
 def test_too_large_request_chunked(dynamodb, test_table):
+    if Version(urllib3.__version__) < Version('1.26'):
+        pytest.skip("urllib3 before 1.26.0 threw broken pipe and did not read response and cause issue #8195. Fixed by pull request urllib3/rllib3#1524")
```

Comment:
Typo in `urllib3/rllib3#1524` (here and below copied):

```suggestion
        pytest.skip("urllib3 before 1.26.0 threw broken pipe and did not read response and cause issue #8195. Fixed by pull request urllib3/urllib3#1524")
```

Reply:
Thanks. Pushed a new version fixing this typo